src/PowerShellEditorServices/Services/TextDocument/Handlers/PsesSemanticTokens.cs

```diff
@@ -43,50 +43,52 @@ public PsesSemanticTokens(ILogger<PsesSemanticTokens> logger, WorkspaceService w
             CancellationToken cancellationToken)
         {
             ScriptFile file = _workspaceService.GetFile(DocumentUri.GetFileSystemPath(identifier));
-            Token[] tokens = file.ScriptTokens;
-            foreach (var token in tokens){
+            foreach (var token in file.ScriptTokens){
```

Comment:
The inferred type here is not explicitly given on the RHS, so it's best not to use `var`

```suggestion
            foreach (Token token in file.ScriptTokens) {
```